mesonbuild/build.py

```diff
@@ -2446,14 +2446,14 @@ def __init__(self,
                  backend: T.Optional['Backend'] = None,
                  ):
         # TODO expose keyword arg to make MachineChoice.HOST configurable
-        super().__init__(name, subdir, subproject, False, MachineChoice.HOST, environment)
+        super().__init__(name, subdir, subproject, False, MachineChoice.HOST, environment,
+                         install, build_always_stale)
```

Comment:
This feels logically like it belongs squashed into the previous commit.

Reply:
done